pimd/pim_vty.c

```diff
@@ -174,7 +174,11 @@ int pim_global_config_write_worker(struct pim_instance *pim, struct vty *vty)
 	int writes = 0;
 	struct pim_ssm *ssm = pim->ssm_info;
 	char spaces[10];
-
+#if PIM_IPV == 4
+	const char *af = "ip";
```

Comment:
Can we move this to a header / `#define PIM_AF_NAME "ip" / "ipv6"` ?

Reply:
Yes good idea. Moved.